### ISSUE TITLE
Please upgrade to null safety for latest flutter: 2.0.0

### ISSUE THREAD

**Issue by gouthamravella** [fluttercommunity/flutter_google_places#149]
Also please upgrade the rxdart ^0.21.0 in the package. Which is failing with the dependency package cached_network_image: >=3.0.0-nullsafety

**Comment by jonasbark** [comment]
I'd actually prefer it would remove the cross dependency to rxdart altogether

**Comment by gianlucainnocente** [comment]
We need the rxdart dependency upgraded

**Comment by hoc081098** [comment]
Workaround:
```yaml
dependency_overrides:
  flutter_google_places:
    git:
      url: https://github.com/hoc081098/flutter_google_places.git
      ref: 0566867ee2abe5c8c2ba180678f18309845b23a6
```

**Comment by harsimranmaan** [comment]
@hoc081098 Do you mind sending your changes upstream and open an PR here?

**Comment by Theunodb** [comment]
Do you have plans to update this?

**Comment by Johan-glitch1412** [comment]
Any updates on this? Need this urgently.

**Comment by Johann-Hagen** [comment]
I urgently need this fixed, my clients are being typical clients about it...

**Comment by Teutonic-Knight-0** [comment]
Any update?

**Comment by jonasbark** [comment]
I'm sure he'd accept pull requests for this - help him if it's that urgent. Otherwise, I think it's impolite to press someone that uses his free time on this.
In the meantime use the fork from @hoc081098

**Comment by hoc081098** [comment]
Just published as new name: https://pub.dev/packages/flutter_google_places_hoc081098. Only replace the importing
```diff
- import 'package:flutter_google_places/flutter_google_places.dart';
+ import 'package:flutter_google_places_hoc081098/flutter_google_places_hoc081098.dart';
```

**Comment by erperejildo** [comment]
Any update guys?

**Comment by KevMorelli** [comment]
Implemented on v0.3.0

**Comment by abdulrehmank7** [comment]
I created a utility class for Google places auto complete it uses null safety: https://arkapp.medium.com/flutter-and-google-map-for-beginners-part-1-fdba4ab22148
